ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -1095,4 +1108,22 @@ public boolean equals(final Object o) {
   public int hashCode() {
     return Objects.hash(forkSchedule);
   }
+
+  public Optional<Integer> getMaxBlobsPerBlockAtSlot(final UInt64 slot) {
+    final SpecVersion specVersion = atSlot(slot);
+    switch (specVersion.getMilestone()) {
+      case DENEB, ELECTRA -> {
+        return Optional.of(
+            specVersion.getConfig().toVersionDeneb().orElseThrow().getMaxBlobsPerBlock());
+      }
+      default -> {
+        if (specVersion.getMilestone().isGreaterThan(CAPELLA)) {
```

Comment:
why not greaterThanOrEqualTo(FULU)? It's a bit confusing with CAPELLA here and takes some cycles to realize DENEB and ELECTRA were already covered

Reply:
because in the set of not deneb/electra is phase0, bellatrix, altair

once the implementation is fully cleaned up with deneb/electra then this becomes a single call to the BPO, but as default returning empty is what i wanted (pre-deneb fall to the final return)

this is all temporary until deneb / electra use bpo